src/codeu/chat/client/core/View.java

```diff
@@ -136,4 +137,30 @@ public View(ConnectionSource source) {
 
     return messages;
   }
+
+
+//Updated for UpTime by Julia 5/22
+  	@Override 
+	public ServerInfo getInfo() {
+		
+		try (final Connection connection = source.connect()) {
+		  
+	    Serializers.INTEGER.write(connection.out(), NetworkCode.SERVER_INFO_REQUEST);
```

Comment:
minor note: you probably want to fix your formatting here. I usually just rely on my IDE's auto-formatter. (In Eclipse, that's ctrl+A to select all, then ctrl+I to auto-indent everything, then ctrl+S to save. I do that about once every 5 seconds, haha)